packages/fast-components-react-msft/src/radio/README.md

```diff
@@ -0,0 +1,8 @@
+# Radio
+*Radio* buttons are almost always listed together in groups.
```

Comment:
Perhaps something like "*Radio* buttons are designed for selecting one value out of a set. Therefore, *radio* buttons are almost always listed together in groups."

Reply:
sounds good